security/s2a/internal/crypter/aesgcm_test.go

```diff
@@ -40,11 +41,11 @@ func getGCMCryptoPair(key []byte, t *testing.T) (s2aAeadCrypter, s2aAeadCrypter)
 
 func isFailure(result string, err error, got, expected []byte) bool {
```

Comment:
If this is going to be used in the ChaChaPoly test, we should move this over the the `wycheproofutil.go` or `testutil/common.go` file

Reply:
That was my thought as well, but when I moved it to common.go, I recieved an interesting error:

`go: finding module for package google.go.lang/grpc/grpc-go/security/s2a/internal/crypter/testutil
common.go:6:2: cannot find module providing package google.go.lang/grpc/grpc-go/security/s2a/internal/crypter/testutil: unrecognized import path "google.go.lang/grpc/grpc-go/security/s2a/internal/crypter/testutil": https fetch: Get "https://google.go.lang/grpc/grpc-go/security/s2a/internal/crypter/testutil?go-get=1": dial tcp: lookup google.go.lang on 127.0.0.1:53: no such host`

It seems that it couldn't locate 'google.go.lang/grpc/grpc-go/security/s2a/internal/crypter/testutil'?

Reply:
`testutil` is already imported in your `chachapoly_test.go` file, so I'm not sure if it's an import issue. When you moved it to `testutil/common.go`, did you export the function?
ie.
`isFailure`->`IsFailure`

I just tried moving the function over to `common.go` and it's working for me.

Reply:
I moved it to wycheproofutil and worked fine.

Reply:
@Ryanfsdf @davisgu Can this be resolved? Or is there anything left unaddressed?

Reply:
I think it just needs to be removed from `aesgcm_test.go` since it was moved to `wycheproofutil.go`

Reply:
^will create a new pr for the aesgcm changes